extentions/gcc.bzl

```diff
@@ -21,9 +21,76 @@ def _gcc_impl(mctx):
         if not mod.is_root:
             fail("Only the root module can use the 'gcc' extension")
 
+    DEFAULT_FEATURES = [
+        "minimal_warnings",
+        "strict_warnings",
+        "treat_warnings_as_errors",
```

Comment:
for now: all minimal_warnings should be errors from day 0. Then we follow up with the others.